bip-emilengler-terminology-for-payment-recipient-identifiers.mediawiki

```diff
@@ -0,0 +1,49 @@
+<pre>
+  BIP: UNDEFINED
+  Title: Terminology for payment recipient identifiers
+  Author: Emil Engler <me@emilengler.com>
+  Status: Draft
+  Type: Informational
+  Created: 2019-10-17
+  License: CC0
+</pre>
+
+==Abstract==
+This BIP proposes a new term for 'address'
+
+==Specification==
+The new term is:
+''Bitcoin'' '''Invoice''' ''Address''
+
+==Motivation==
+Bitcoin addresses are intended to be only used '''once''' and you should generate a new one for every new incoming payment.
+The term 'address' however indicates consistency because nearly everything on the internet or the offline world with the term 'address'
+is something that rarely or even never changes (postal address, e-mail address, IP-Addresses (depends heavily on the provider), etc.)
+The motivation for this BIP is to change the term address to something that indicates that the address is connected to a single transaction.
+
+==Rationale==
+The reason why we use ''Bitcoin Invoice Address'' or just ''Invoice'' is to show that it is more like a temporary thing.
```

Comment:
The "temporary" seems a bit off. It's not really temporary, an invoice doesn't change or go away. It's just that it is single-use.
Therefore, how about "[…]is to emphasize that it is single-use."

Reply:
Single use is what I meant.
Thanks for the word(s) suggestion